### ISSUE TITLE
Downsample images uploaded for 'Student Photos'

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#3724]
**Describe the bug**
If an instructor uploads very large images, the page cannot handle displaying them.
Let's downsample large images at upload time.

Target size might be under 100K?  (RPI student id photos are about 12K,  exact size can be negotiated)

**Comment by MasterOdin** [comment]
Given that an instructor can upload the image manually through ssh, we probably want to also either:
1. disregard very large images
2. downsample them if we detect that they're large, overwriting the original and using that new image going forward.

@bmcutler your opinion on which should be done?

**Comment by bmcutler** [comment]
In my original comment I was definitely asking for #2 please.

**Comment by bmcutler** [comment]
Oh, sorry I realize you said ssh.  Let's not do anything with files that appear in that directory.  Let's assume the instructor knows best.  Eventually instructors may not have ssh access.

**Comment by MasterOdin** [comment]
Yeah, sorry, yeah, I meant for uploading pictures through SSH and then how to handle when we go to show these images. I'm fine with the strategy of assuming instructors know best.

How do you want to handle existing classes that might have jumbo images for their student photos?

**Comment by bmcutler** [comment]
Don't edit any files that are already saved.  It's fine if the page just doesn't work right, or it's fine to skip/omit the large image (hopefully instead pasting a simple text message at that location e.g. 'image to large')
